tests/fixers/test_django_urls.py

```diff
@@ -213,6 +213,39 @@ def test_re_path_unanchored_end():
     )
 
 
+def test_re_path_unanchored_end_with_include():
```

Comment:
This end is anchored
```suggestion
def test_re_path_anchored_end_with_include():
```